editor/src/components/canvas/ui-jsx-canvas.test-utils.tsx

```diff
@@ -156,14 +155,9 @@ export function renderCanvasReturnResultAndError(
   }
   const updatedContents = contentsToTree(projectContents)
 
-  const baseRequireFn = getRequireFn(NO_OP, updatedContents, {}, {}, 'canvas')
-  const requireFn: UiJsxCanvasProps['requireFn'] = (importOrigin: string, toImport: string) => {
-    switch (toImport) {
-      // here we can manually insert extra dependencies, such as antd
-      default:
-        return baseRequireFn(importOrigin, toImport)
-    }
-  }
+  const curriedRequireFn = (innerProjectContents: ProjectContentTreeRoot) =>
```

Comment:
I don't think I'd refer to this as curried because the result of calling this isn't itself curried. Maybe `requireFnFromProjectContents`?

Reply:
Yep that's fair, I'll come back and rename this